lib/decoder/standard.js

```diff
@@ -74,7 +74,7 @@ exports.decode = async function (buffer) {
   const le = url.parse('?' + (data['cs-uri-query'] || ''), true).query.le
 
   // uri stem does not include query params (ex: /123/<guid>/<digest>/file.mp3)
-  const digest = (data['cs-uri-stem'] || '').split('/').slice(-2, -1)[0]
+  const digest = (data['cs-uri-stem'] || '').replace(/\/+$/, '').split('/').slice(-2, -1)[0]
```

Comment:
Standard S3 cloudfront logs don't have the query param in this part.  So just replace trailing `/`.

Reply:
Looks like that is part of the sec - this is supposed to have no qs:
`
This field MUST specify the URL actually used by the [client](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-wmlog/628f5c1d-317e-478f-87d0-6fe2df477402#gt_60e0e1fa-66fe-41e1-b5e3-ceab97e53506).
Any query strings MUST be excluded from the URL. (This means that the value of the cs-uri-stem field is equal to the URL actually used by the client, truncated at the first "?" character.)
`
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-wmlog/fc4b49d7-e83f-4389-8063-414f8bad80dd